src/main/resources/reference-data/sscs-venues.csv

```diff
@@ -68,7 +68,7 @@
 228,SC228,SSCS Newcastle,Manorview House,Kings Manor,,Newcastle,Tyne and Wear,NE1 6PA,0191 2012300,4401,"https://www.google.com/maps/place/Manorview+House,+HMCTS+Tribunal+Centre/@54.9734558,-1.6082254,17z/data=!3m1!4b1!4m5!3m4!1s0x487e70c7e485365f:0x1cc41e14d5597a9f!8m2!3d54.9734527!4d-1.6060367"
 232,SC232,SSCS Newcastle,Kings Court,Royal Quays,Earl Grey Way,,North Shields,NE29 6AR,0191 2960249,4401,"https://www.google.com/maps/place/Kings+Court,+Earl+Grey+Way,+North+Shields+NE29+6AR,+UK/@54.9984312,-1.4625101,17z/data=!3m1!4b1!4m5!3m4!1s0x487e6e551931a0f1:0x7e97a5ac38f9cacb!8m2!3d54.9984003!4d-1.4601734"
 233,SC233,SSCS Newcastle,South Tyneside Magistrates Court,Millbank/Secretan Way,,South Shields,Tyne & Wear,NE33 1RG,,4401,"https://www.google.co.uk/maps/place/South+Tyneside+Magistrates+Court/@54.9927658,-1.441352,17z/data=!4m13!1m7!3m6!1s0x487e6f1c44bfadef:0x8944659d4eb2dcb5!2sSouth+Shields+NE33+1RG!3b1!8m2!3d54.9930239!4d-1.439086!3m4!1s0x487e6f1c44bbd355:0x363f29882357e1db!8m2!3d54.9932477!4d-1.4390514"
-236,SC236,SSCS Newcastle,Hylton Park,"Unit 2c, Hylton Park",Wessington Way,Sunderland,Tyne and Wear,SR5 3HD,0191 5102729,4401,"https://www.google.com/maps/place/Sunderland+SR5+3HD,+UK/@54.9185944,-1.4243795,18.5z/data=!4m5!3m4!1s0x487e65c86b19d583:0xf728568b327626b1!8m2!3d54.9186452!4d-1.4240203"
+236,SC236,SSCS Newcastle,Sunderland Courts & Tribunals,Civil and Tribunal Entrance,Gillbridge Avenue,Sunderland,Tyne and Wear,SR1 3AP,0191 5102729,4401,"https://www.google.co.uk/maps/place/Sunderland+Magistrates'+Court/@54.9072103,-1.3900305,17z"
```

Comment:
the url "https://www.google.co.uk/maps/place/Sunderland+Magistrates'+Court/@54.9072103,-1.3900305,17z" does not work as expected